homeassistant/components/hassio/addon_manager.py

```diff
@@ -65,6 +65,14 @@ async def wrapper(
     return handle_hassio_api_error
 
 
+@dataclass
+class AddonDiscoveryInfo:
+    """Represent the add-on discovery info."""
+
+    config: dict
+    uuid: str
```

Comment:
There are two more fields, `addon` and `service`, should we add them too for completeness? :thinking: 

https://developers.home-assistant.io/docs/api/supervisor/models#discovery

Reply:
This is no longer relevant